png-afl/Cargo.toml

```diff
@@ -1,14 +1,10 @@
 [package]
 name = "png-afl"
-version = "0.1.0"
-authors = ["nwin <nwin@users.noreply.github.com>"]
+version = "0.2.0"
+authors = ["Sergey Davidoff <shnatsel@gmail.com>", "Paul Grandperrin <paul.grandperrin@gmail.com>"]
```

Comment:
I think nwin removed as author here by accident.

Reply:
I have entirely replaced the afl/ subfolder without using anything that's in there previously. Its current incarnation is based only on https://github.com/rust-fuzz/targets, which is why Paul Grandperrin is credited.

I agree nwin should be credited for his contribution in some way, but I have removed him from the copyright notice for the source files the current afl/ folder is not based on his work in any way, and such misattribution is deliberately prohibited under some licenses, e.g. some BSD variants. This crate is dual-licensed under MIT and Apache, which do not have such a clause, so crediting nwin here would not be in conflict with the license.

I will re-add him as an author if you believe that is the best way to go about it.

Reply:
Ah, it's under the "png-afl" directory, didn't saw that.

@nwin OK with this?

Reply:
Should I split the panic fixes to a separate PR so they would not be held up by a copyright notice?

Reply:
I think it's OK. @nwin can just add himself back as author if he disagrees.